### PR TITLE
Upgrade to kvm-bindings v0.4.0-1

### PR DESCRIPTION
**Pull request by alindima**
*Description of changes:*

Cherry-picked changes from upstream and released v0.4.0-1.

The highlight of this is the upgrade of `vmm-sys-util` to v0.8.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by acatangiu — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ kvm-v4_20_0 = []
 fam-wrappers = ["vmm-sys-util"]
 
 [dependencies]
-vmm-sys-util = { version = ">=0.2.0", optional = true }
+vmm-sys-util = { version = ">=0.8.0", optional = true }
 
 versionize = { version = ">=0.1.4" }
```
Comment (sandreim) [inline Cargo.toml:19]: Should we bump this due to vmm-sys-util version upgrade ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.4.0-1]
+
+- [upstream] vmm-sys-utils dependency bumped to match kvm-ioctls
```
Comment (sandreim) [inline CHANGELOG.md:5]: Can we be specific - 0.8.0 ?

File: Cargo.toml (hunk 1 above)
Comment (alindima) [reply Cargo.toml]: done

File: CHANGELOG.md (hunk 2 above)
Comment (alindima) [reply CHANGELOG.md]: done

**Review by acatangiu — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

**Pull request merged**